### PR TITLE
Rework content management guide

### PR DESCRIPTION
**Pull request by maximiliankolb**
* Set headings to level 1
* Add anchors
* Remove unnecessary whitespace
* Use 'RHEL' attribute
* Make anchors and headings more downstream friendly
* Replace 'capsule' with 'Proxy'
* Move 'appendix' directive to parent pages

In a follow up PR, I want to split up content into different files. This will result in one heading per file.

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Rebased to master.

**Comment by melcorr** [comment]
This is on my to do list for tomorrow morning!

**Comment by lzap** [comment]
I assigned you if you dont mind, I am assuming you can finish this. Ping katello guys in case of technical questions.

**Review by melcorr — CHANGES_REQUESTED** [review]
Hey @maximiliankolb 
Some small stuff for now. 
Have asked @bbuckingham to take a look at one section or point me to an engineer who can help clarify. 
I appreciate you helping to make this more consistent. 
There might be comments from others too so feel free to wait a few days before applying feedback to save yourself some effort.

File: guides/doc-Content_Management_Guide/topics/Containers_interacting_with_registry.adoc [hunk 1]
```diff
@@ -3,32 +3,36 @@
 
 Podman and Docker can be used to fetch content from Katello container registries.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/doc-Content_Management_Guide/topics/Containers_interacting_with_registry.adoc:4]: Applied.

File: guides/doc-Content_Management_Guide/topics/Containers_interacting_with_registry.adoc [hunk 2]
```diff
@@ -3,32 +3,36 @@
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/doc-Content_Management_Guide/topics/Containers_interacting_with_registry.adoc:3]: Removed.

File: guides/doc-Content_Management_Guide/topics/Containers_interacting_with_registry.adoc [hunk 3]
```diff
@@ -3,32 +3,36 @@
 
 Podman and Docker can be used to fetch content from Katello container registries.
 
-.Container Registries on Smart Proxies
+.Container Registries on {SmartProxies}
 
-On {SmartProxies} with content, the https://github.com/Katello/smart_proxy_container_gateway[Container Gateway] {SmartProxy} plugin acts as the container registry.  It caches authentication information from Katello and proxies incoming requests to Pulp.  The Container Gateway comes default on {SmartProxies} with content.
+On {SmartProxies} with content, the https://github.com/Katello/smart_proxy_container_gateway[Container Gateway] {SmartProxy} plugin acts as the container registry.
+It caches authentication information from Katello and proxies incoming requests to Pulp.
+The Container Gateway comes default on {SmartProxies} with content.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/doc-Content_Management_Guide/topics/Containers_interacting_with_registry.adoc:10]: Applied.

**Comment by maximiliankolb** [comment]
Rebased to master and applied your suggestions.

I would appreciate if we could move forward, so I can work on splitting up file for individual procedures. I want to avoid rebase hell^^. 

Nevertheless, I appreciate any suggestions and would **love** to work on them **afterwards**.

We can squash before merging, obviously.

**Comment by melcorr** [comment]
@maximiliankolb sure. I can talk with @bbuckingham and raise a separate PR.

**Comment by maximiliankolb** [comment]
Much appreciated @melcorr

**Pull request merged**